shell/src/main/java/org/crsh/plugin/ResourceManager.java

```diff
@@ -165,7 +166,7 @@ void refresh() {
   }
 
   /** . */
-  private static final byte[] SEPARATOR = System.getProperty("line.separator").getBytes();
+  private static final byte[] SEPARATOR = System.getProperty("line.separator").getBytes(Charset.forName("UTF-8"));
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.